### ISSUE TITLE
Can't remove backups

### ISSUE THREAD

**Issue by snippem** [home-assistant/supervisor#3084]
### Describe the issue you are experiencing

Since the latest update of the supervisor i can't delete backups.
It is not a big deal because i can still remove them with samba but it ui wise it does not work.
i have minimal logs:

 

### What is the used version of the Supervisor?

supervisor-2021.08.0

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### What is the version of your installed operating system?

Home Assistant OS 6.2

### What version of Home Assistant Core is installed?

core-2021.9.0b2

### Steps to reproduce the issue

1.make backup full
2.tried to delete 
3.did not delete says check logs
...

### Anything in the Supervisor logs that might be useful for us?

```txt
log supervisor

21-08-28 11:13:45 INFO (MainThread) [supervisor.backups.manager] Found 1 backup files
21-08-28 11:13:45 INFO (MainThread) [supervisor.backups.manager] Found 1 backup files

log core:

2021-08-28 11:13:50 ERROR (MainThread) [homeassistant.components.hassio.handler] /backups/52a97a9b/remove return code 404
2021-08-28 11:13:50 ERROR (MainThread) [homeassistant.components.hassio] Failed to to call /backups/52a97a9b/remove -
```

### Additional information

none

**Comment by ludeeus** [comment]
This was fixed in #3083 👍

**Comment by snippem** [comment]
Okay waiting for the release :-) thanks for the fast fix.
